fieldservice_mobile/views/fsm_order_view.xml

```diff
@@ -28,4 +28,18 @@
         </field>
     </record>
 
+    <record id="fsm_order_artisent_changes_inherit model" model="ir.ui.view">
+        <field name="name">fsm.order.artisent.changes.inherit</field>
+        <field name="model">fsm.order</field>
+        <field name="inherit_id" ref="artisent.fsm_order_artisent_changes"/>
```

Comment:
@ReedHayashikawa Remove the artisent module dependency here and get the latest 12.0 branch before pushing your latest code changes related to this ticket. Thanks